pkgs/by-name/mm/mmtui/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  gitUpdater,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mmtui";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "SL-RU";
+    repo = "mmtui";
+    tag = "v${version}";
+    hash = "sha256-s+50kz6OODZ0xKz8oNF2YEzk+mLZ6gXXynl8g6Uwdo4=";
+  };
+
+  cargoHash = "sha256-PQaS5aXG6MCVOKfM6Y7dvAd8BA/8BJQvnXTgitMwtf8=";
```

Comment:
```suggestion
  cargoHash = "sha256-9F1YMepkWksTQRrkziNhLxVJnhoDH17lSKef5kOjp3Y=";
  useFetchCargoVendor = true;
```

Reply:
Currently a treewide migration is being carried out in order to use fetchCargoVendor everywhere and eventually set this as the new default